Extractor/Nodes/UAVariable.cs

```diff
@@ -552,7 +553,8 @@ public SourcedNodeWrite<CogniteExtractorTimeSeries> ToIdmTimeSeries(
                     if (prop is not UAVariable propVar) continue;
                     if (metaMap.TryGetValue(prop.Name ?? "", out var mapped))
                     {
-                        var value = client.StringConverter.ConvertToString(propVar.Value, propVar.FullAttributes.DataType.EnumValues);
+                        if (propVar.Value == null) continue;
```

Comment:
same comment here (could you explain for me as an outsider, why this was not skipped before?)